features/analytics.feature

```diff
@@ -4,13 +4,20 @@ Feature: Analytics
   I want to be able to track user behaviour
 
   @javascript
-  Scenario: Ecommerce tracking
+  Scenario: eCommerce tracking
     When I view the all content finder
     Then the ecommerce tracking tags are present
     And I search for lunch
     And I submit the form
     Then the data-search-query has been updated to lunch
 
+@javascript
+  Scenario: Zero results sends eCommerce tracking
+    When I view the all content finder
+    And I search for superted
```

Comment:
why don't we have any results for superted? 😟

Reply:
no one cares about discarded teddies in corners of storerooms anymore